---
templateKey: about-page
path: /about
title: A collaborative effort
---
## COVID-19 host genetics initiative

### Content

**\- Overview**\
**\- Aims**\
**\- Principle of collaboration**\
**\- Approach**\
&nbsp;&nbsp;&nbsp;&nbsp;+ A: Communication and sharing of resources\
&nbsp;&nbsp;&nbsp;&nbsp;+ B: Analysis\
&nbsp;&nbsp;&nbsp;&nbsp;+ C: Results and data sharing\
**\- Terms of use of results and Authorship**\
**\- Support for genotyping and sequencing** <br/><br/>

### Overview

The COVID-19 pandemic is a global crisis creating severe disruptions across the economy and health system. Insights into how to better understand and treat COVID-19 are desperately needed. Given the importance and urgency in obtaining these insights, it is critical for the scientific community to come together around this shared purpose.

*The COVID-19 host genetics initiative* brings together the human genetics community to generate, share, and analyze data to learn the genetic determinants of COVID-19 susceptibility, severity, and outcomes. Such discoveries could help to generate hypotheses for drug repurposing, identify individuals at unusually high or low risk, and contribute to global knowledge of the biology of SARS-CoV-2 infection and disease.

*Nothing is written in stone other than we must all act together and with no personal gain or ownership of results – just rapid and immediate dissemination of the maximum possible data and information that can be responsibly released.*

### Aims

*The COVID-19 host genetics initiative* is a bottom-up collaborative effort that has three main goals:

1. Provide an environment to foster the sharing of resources to facilitate COVID-19 host genetics research (e.g. protocols, questionnaires).
2. Organize analytical activities across studies to identify genetic determinants of COVID-19 susceptibility and severity.
3. Provide a platform to share the results from such activities, as well as the individual-level data where possible, to benefit the broader scientific community.

### Principles of collaboration

*7 Principles of Collaboration*:

1. Collaborate in an environment of honesty, fairness, and trust.
2. Promote early-career researchers.
3. Respect other groups’ data.
4. Operate transparently with a goal of no surprises.
5. Seek permission from each group to use results prior to public release.
6. Do not share another group’s results with other parties without permission.
7. We should not inhibit any work being done within any individual studies (or between pairs of studies).

### Approach

#### A: Communication and sharing of resources

Most of the current conversation between participants is happening on the ICDA slack channel. We aim to maintain this channel as the main avenue to exchange information.

Resources that are available and should continue to be shared across participants:

* [Data dictionary](https://docs.google.com/spreadsheets/d/1RXrJIzHKkyB8qx5tHLQjcBioiDAOrQ3odAuqMS3pUUI/edit#gid=549383528)
* [Questionnaires](https://docs.google.com/document/d/12O6h5EcVCb7y3w8vJPEef1Tpjg0x2fmge9c1uhYTlfo/edit)
* [Consent/protocols](https://drive.google.com/drive/folders/101UkwCExOS5T-8ZhioCmEwLjDuywZiOl)
* [Register of participant studies](https://www.covid19hg.org/partners/). Registration can be done [here](https://www.covid19hg.org/register/).

#### B: Analysis

#### B:1 Data resources – Retrospective studies/existing biobanks

From biobanks with existing significant genetic data and active connections to health systems, there is an opportunity to opportunistically and rapidly develop a genetic study on susceptibility and severity. For this group of studies we will use the structure that has already been put in place by the [Global Biobank Meta-analysis Initiative](https://www.globalbiobankmeta.org/).

We don’t expect that this group of studies will be able to share individual-level data and is likely that the analyses will be done by each participant center and meta-analyzed. Such efforts may happen quickly with relatively small numbers of cases so we encourage all to update case numbers in existing biobank studies, and when available summary statistics from analyses completed, as soon as they are available and aim to update weekly if possible.

#### B:2  Data resources – Prospective studies

Understanding that the opportunistic approach (B:1) will be limited in numbers in most locations, in parallel many have started directly consenting incoming COVID-19 patients with a new protocol in order to build the numbers more rapidly. In some cases, this may take place in the context of a biobank consent, in some places this will require a specific new study. More than just the critical jump in scale for studying progression, severity, and outcomes, these studies bring important additional opportunities not only for deeper DNA studies, but potentially informative viral and antibody profiling and epitope mapping experiments which should be defined and implemented in many sites with relatively small blood/plasma requirements. For this group of studies, the COVID-19 host genetics initiative can provide support with protocols and questionnaires, as described in (A). We strongly encourage these new studies to include in the informed consent a sentence to inform the participants that data will be shared with other scientists to advance COVID-19 research.

Every group is free to choose a study design that suits the study-specific research question and to pursue different approaches for genotyping/sequencing.

We expect that many of the studies will pull individual-level genetic and clinical data together to advance analysis beyond simple GWAS, but also to allow other researchers outside the initiative to use the data. Plans for data sharing are described in (C).

#### B:3 Analytical strategy

We define two approaches in regards to the analytical activities.

##### Centralized analytical activities

These simple analyses (e.g. SNP-based association analysis, gene-burden test) focuses on some key phenotypes that can benefit from maximal sample size. Invitation to participate in the analysis is sent out on a regular basis to all the partners.

* We defined a set of key initial **phenotypes** that could be analysed across groups,  available [here](https://docs.google.com/document/d/1okamrqYmJfa35ClLvCt_vEe4PkvrTwggHq7T3jbeyCI/edit?usp=sharing).
* We have drafted an **analysis plan** available [here](https://docs.google.com/document/d/16ethjgi4MzlQeO0KAW_yDYyUHdB9kKbtfuGW4XYVKQg/edit).
* **Meta-analysis results** are made available as soon as generated on the website [here](https://www.covid19hg.org/results/).
* **In-silico follow-up results** are  generated by the cummunity and made available [here](https://www.covid19hg.org/blog/2020-06-29-in-silico-follow-up-results/).

##### Additional analyses that require specialised approaches or focus on specific patients groups

*The COVID-19 host genetics initiative* provides a platform for groups to self-assemble in working groups. The suggestion is to work together, create a slack channel and an analysis plan. Example are the `#ardsgwas`, `#covid19-hg-clonalhematopoeisis` Slack channel. *The COVID-19 host genetics initiative* helps to sponsor these efforts and connect researchers interested in similar topics. The main efforts are described [here](https://www.covid19hg.org/projects/).

#### C: Results and data sharing

All participants in the initiative will collaborate following the principle of collaboration outlined in the *principle of collaboration* section. Studies are invited to share individual-level data or results summary statistics. More information about data sharing is provided [here](https://www.covid19hg.org/data-sharing/).

Following the analysis described in the analysis plan, we run periodical iterations of the analysis and make the summary statistics results publicly available via an association browser which will be hosted on the website [here](https://www.covid19hg.org/results/).

The code to perform the analysis will also be made publicly available [here](https://github.com/covid19-hg)

### Terms of use of results and authorship

Accelerating discovery, results from meta-analyses generated by the COVID-19 Host Genetics Initiative are [made public immediately](https://www.covid19hg.org/results/) for the benefit of the wider biomedical community to advance discovery.

We envision two main scenarios for the use of the meta-analyses data with different suggested authorship guidelines.

* Groups using the results as a jumping-off point or supportive evidence for functional studies or in silico analyses are encouraged to acknowledge COVID-19 HGI but are not required to include authorship unless they feel the contribution warranted it.
* Groups conducting analyses that might overlap with the core discovery efforts of the consortium are encouraged to contact us and join, rather than compete, with these efforts to accelerate discovery. For example, if a research group aims to perform and publish larger genome-wide discovery analyses including pre-publication summary statistics released by COVID-19 HGI, we would request that such groups contact us in advance to discuss fair authorship considerations which respect the many groups that have shared data to perform such analyses together and with immediate public release. We will advocate for including the list of people included in the [acknowledgment page](https://www.covid19hg.org/acknowledgements/) as co-authors.

Whatever the activity, we request that all researchers performing additional research of any kind utilizing these results to share their findings publicly in the spirit of providing the broadest access to emerging knowledge surrounding SARS-CoV-2 infection.  We encourage all groups performing genetic studies to share their results and contribute to the design and execution of meta-analyses together

### Support for genotyping and sequencing

There are several interested parties offering to make additional contributions to this effort in terms of genotyping/sequencing:

* **FIMM**, University of Helsinki offered to perform Illumina GSA genotyping at no cost for those without resources or facilities. Contact: Andrea Ganna.
* The Human Genomics Facility (HuGe-F) of the Department of Internal Medicine at **Erasmus MC** offers free of charge genotyping of proven COVID-19 cases, using Illumina GSA or Affymetrix /Thermofisher PMDA. Genotype data of these COVID-19 cases will be used to support the COVID-19 host genetics initiative. HuGe-F will be processing consented DNA samples derived from blood and nasal swabs (used for virus detection). For more information, please visit [www.glimdna.org](http://www.glimdna.org).
* **Regeneron** (relayed by Goncalo Abecasis) offered to perform genotyping and/or WES on samples in which there is a commitment to rapid data sharing.
* **Illumina** representatives in Europe, the Americas and Asia Pacific are exploring ways to contribute reduced price reagents and arrays to this effort. Contact: Markus Boehm for Illumina Europe projects: [mboehm@illumina.com](mailto:mboehm@illumina.com); Jay Kaufman for Illumina Americas projects: [jkaufman@illumina.com](mailto:jkaufman@illumina.com); Tamsin Eades for Illumina Asia Pacific projects: [teades@illumina.com](mailto:teades@illumina.com).
* **Thermo Fisher** representatives in Europe are exploring ways to contribute reduced price reagents and arrays to this effort. Thermo Fisher offers discounted pricing on the new Axiom Human Genotyping SARS-Cov-2 research array in addition to their Axiom Precision Medicine and UK Biobank Axiom array. Contact: [shantanu.kaushikkar@thermofisher.com](mailto:shantanu.kaushikkar@thermofisher.com).
* **Helix** can perform discounted Exome+ (clinical grade exome + genotyping backbone equivalent to GSA) sequencing for any COVID-19 samples where the data will be made rapidly available. Contact: [Liz Cirulli](mailto:liz.cirulli@helix.com).
* **Gencove** can offer discounted or no cost low-pass WGS for samples with rapid data sharing. Contact: Joe Pickrell.
* **Genuity Science** is a contract genomics and data-sourcing, analytics and insights organization headquartered in Boston, Massachusetts, USA with offices in Dublin, Ireland and Reykjavik, Iceland. The company is offering gratis generation of genomic research datasets on the basis that the generated datasets are made available in a public archive as quickly as possible. Contact: Tom O’Dwyer.
* The National Genomics Infrastructure (NGI) at **SciLifeLab** in Sweden will provide heavily discounted genotyping with Illumina GSA-MD for COVID-19 patient samples. We also provide services for WES, WGS, RNA-sequencing, etc. NGI prioritizes Swedish PIs, but can take projects from other countries on a case by case basis. Contact: [support@ngisweden.se](mailto:support@ngisweden.se).
* **Dante Labs** can perform heavily discounted clinical grade WGS and WES as well as RNAseq on Illumina platforms for any sample of patients affected by Covid-19. Data will be made available in less than three weeks only. The lab also performs RT-qPCR analysis for virus detection. Contact: [alessandro.baravelli@dantelabs.com](mailto:alessandro.baravelli@dantelabs.com).
* **NIH** is making a series of commitments to the identification of genetic determinants associated with COVID-19 outcomes and susceptibility and is supporting genotyping/sequencing efforts. More information can be found [here](https://docs.google.com/document/d/1tI0QFTDLPkUtzj3jeRJMGKNsbkopouxE3fVIXUxpWVs/edit?usp=sharing).