---
templateKey: blog-post
title: In-silico follow-up results
date: 2020-06-29T13:17:16.411Z
featuredpost: true
featuredimage: ''
tags:
  - '#in-silico'
type: blog-post
---
# In-silico follow-up results

Here we  post the results from the in-silico follow up analysis done of summary statistics from release 3.

## Heritability and genetic correlations

**Summary:**

**Authors:**

**Results available here**

## TWAS

**Summary:**

**Authors:**

**Results available here**

## **Fine Mapping**

**Summary:**

**Authors:**

**Results available here**



## Summary-data-based Mendelian Randomization

**Summary:** Summary-data based Mendelian Randomization (SMR) analyses of the COVID-19 GWAS summary statistics (ANA_C2_V2).

This analysis aims at detecting colocalisation of GWAS signal (p<1e-5) with methylation and expression quantitative traits loci (mQTL and eQTL respectly) from 11 mQTL and eQTL datasets (ref:<https://cnsgenomics.com/software/smr/#DataResource>). Top SMR results (p<1e-5) are reported. This archive contains two README files (\*.txt and \*.docx format), 2 SMR plots and a result file. Columns of the result file are described here:<https://cnsgenomics.com/software/smr/#SMR&HEIDIanalysis>.

**Authors:** Loic Yengo, Peter Visscher & Naomi Wray

**Results available [here](https://console.cloud.google.com/storage/browser/covid19-hg-in-silico-followup/SMR/?project=covid-19-hg&authuser=0&organizationId=548622027621)**

## PheWAS

**Summary:** PheWAS analysis in the UK Biobank using GP and Hospital Episode Statistics (HES) data. Genetic Score based on COVID-19 associated SNPs. LD clumping was performed at SNPs with Pvalue<10-5, r2=0.1. No outcome passed Bonferroni correction.

**Authors:** Areti Papadopoulou -Eirini Marouli

**Results available [here](https://console.cloud.google.com/storage/browser/covid19-hg-in-silico-followup/phewas/?project=covid-19-hg&authuser=0&organizationId=548622027621)**



## S-PrediXcan/MetaXcan

**Summary:**

**Authors:**

**Results available here**