---
templateKey: blog-post
title: Genetic and non-genetic causes of severe COVID-19
date: 2021-02-19
tags:
  - COVID19
  - genetics
  - biomedical research
  - scicomm
  - litereature review
type: blog-post
---

Written by Nirmal Vadgama, PhD

Edited by Kumar Veerapen, PhD and Jamal Nasir, PhD

*Note: This blog post is catered to the lay audience. The COVID-19 Host Genetics Initiative (HGI) represents a consortium of over 1000 scientists from over 54 countries working collaboratively to share data, ideas, recruit patients and disseminate our findings. For a primer on our study design or the results from the November 2020 (data freeze 4), please read our [blog post](https://www.covid19hg.org/blog/2020-11-24-covid-19-hgi-results-for-data-freeze-4-october-2020/). Our research is iterative, and we summarize our new results via [blog posts](/blog/) and on the [results](/results/r5/) section of our website. Finally, if any vocabulary here is unfamiliar, please send us an email at [hgi-faq@icda.bio](https://www.covid19hg.org/blog/2020-09-24-freeze-3-results/hgi-faq@icda.bio)—we’d be happy to update the information here to provide more clarity. In the coming weeks, additional information explaining concepts or terminology will be made available. In the interim, take a look at [this](https://medlineplus.gov/genetics/understanding/) resource to review the basics of genetics.*

## A pandemic in the genomic era

The 2019 novel coronavirus disease (COVID-19) was declared a pandemic in March 2020 by the World Health Organisation (WHO). It has, to date, resulted in over 110 million cases and 2.43 million deaths globally ([https://covid19.who.int/](https://covid19.who.int/)). Clinically, older adults with existing morbidities, such as heart or lung disease, are at higher risk for developing more serious complications from COVID-19.

The culprit (i.e. virus) causing this deadly disease has been named SARS-CoV-2, which stands for “*severe acute respiratory syndrome coronavirus 2*”. The SARS-CoV-2 virus is the seventh coronavirus species that causes respiratory illnesses in humans ([Jiang, Du and Shi, 2020](https://pubmed.ncbi.nlm.nih.gov/32005086/)). Coronaviruses usually cause mild to moderate upper-respiratory tract illnesses, like the common cold, and utilise similar mechanisms to gain entry into the host cell. This is illustrated in Figure  1. Other coronavirus species that have caused deadly disease outbreaks include SARS (severe acute respiratory syndrome) and MERS (Middle East respiratory syndrome) in 2003 and 2012, respectively.


![Cellular invasion of SARS-CoV-2](/img/unnamed.png)

**Figure 1. Cellular invasion of SARS-CoV-2.**  The Spike or 'S' protein on the surface of the spherical viral particles bind to specific receptors on the surface of their host cells. The S protein of SARS-CoV-2 binds to the human ACE2 receptor and gains entry into the cell with the help of other proteins, including TMPRSS2, NRP1 and Furin.

The outbreak of COVID-19 was first reported in Wuhan, China, and has since spread across the globe by human-to-human transmission. All  available  evidence  suggests  that  SARS-CoV-2  has  a  [zoonotic](https://en.wikipedia.org/wiki/Zoonosis#:~:text=A%20zoonosis%20(plural%20zoonoses%2C%20or,a%20vertebrate)%20to%20a%20human.) source, making it yet another coronavirus to have emerged from animal reservoirs. This new coronavirus causes serious respiratory diseases and presents an urgent need for research to help mitigate this health threat.

There are many variables to consider when unraveling the complex ways in which SARS-CoV-2 interacts with human immune systems. Our genes, route of exposure, viral load, mutations in the virus and our immunological history can produce outcomes ranging from asymptomatic infection to death.

The ongoing concerted international effort is gradually unravelling the role of host genetic factors in COVID-19 susceptibility and severity. The genomics revolution has changed the way we treat and control infectious diseases, from the development of vaccines, to preventing the spread of pathogens, to understanding their origins and evolution.



## Variability in severity

A typical feature of human infections is that not all individuals develop clinical disease, and for those who are infected, there is variability in severity ([Kenney et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28853921/)). The COVID-19 pandemic is no different: there is wide variability in disease onset, clinical symptoms, severity, and prognosis. According to several studies, around 80% of COVID-19 patients were classified as mild (did not develop pneumonia or had a mild case of it), while up to 20% progressed to critical stage with likely death of respiratory failure ([Fu et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32283155/)) (Figure 2).

![Variability of symptoms in COVID-19 patients](/img/litreview_figure2.png)

**Figure 2. Variability of symptoms in COVID-19 patients.**  Figure created by Sophie Limou, PhD


Compared to mild or asymptomatic patients, those who were severely affected had more abnormal laboratory findings including elevated levels of infection-related biomarkers (e.g., C-reactive protein and white blood cells) ([Qin et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32161940/)). In a study investigating the transmission potential of asymptomatic carriers, family members who lived together acquired severe COVID-19 symptoms, such as pneumonia ([Hu et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32146694/)). As these individuals had the same or similar environment, the observed clinical variability might be explained by genetic factors.


## Clues in the genes

Studies investigating the host responses to other infectious diseases, such as SARS-CoV, HIV, influenza, and several other common viruses, have indicated that the host genetic background plays an essential role ([Wang et al., 2020](https://pubmed.ncbi.nlm.nih.gov/33298875/)) (Figure 3).


![Gene categories involved in infectious diseases](/img/litReview_figure3.png)

**Figure 3. Gene categories involved in infectious diseases.** Variations in many genes are associated with infectious disease severity and susceptibility. Each gene encodes for a protein that carries out its distinct cellular function (labelled in bold). Abbreviation: Th, T helper. (Image from [Kenney et al., (2017)](https://pubmed.ncbi.nlm.nih.gov/28853921/)).


Identical twins are particularly useful in genetic studies as they are perfectly matched for genetic and familial background. Discordant identical twins, where only one twin has a disease, are often used to investigate which non-shared environmental factors are influencing the disease in question. A recent UK-based twin study of over 2,600 participants found that genetics determined about 50% of COVID-19 symptoms ([Williams et al., 2020](https://www.medrxiv.org/content/10.1101/2020.04.22.20072124v2)). This suggests that the clinical variability seen in patients may be largely explained by genetic factors.

In August 2020, researchers conducted a genome-wide association study (GWAS) among 2173 COVID-19 patients with local control populations. One region they identified is the locus that encodes blood type. They found that blood group A had a significantly higher risk of infection, while blood group O had a protective effect ([Zhao et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32750119/)).  This is also supported by more recent genetic studies ([Severe Covid-19 GWAS Group, 2020](https://pubmed.ncbi.nlm.nih.gov/32558485/)).

Furthermore, a study of intensive care patients has identified additional genes involved in antiviral immune responses and inflammation. Pairo-Castineira and colleagues used DNA samples from over 2,200 patients with severe COVID-19 who were treated in over 200 intensive care units in the UK ([Pairo-Castineira et al., 2020](https://pubmed.ncbi.nlm.nih.gov/33307546/)).

In this study, the authors identified eight genetic sequences that were more common in patients with the severe form of the infection. These sequences involved genes related to the inflammatory response that is triggered in the presence of a viral attack. For example, the chr12q24.13 variant is linked to genes activating the antiviral response (*OAS1, OAS2, OAS3*); chr19p13.2 variant  is close to the gene that encodes the enzyme tyrosine kinase 2 (*TYK2*); chr19p13.3 encodes for dipeptidyl peptidase 9 (*DPP9*) and chr21q22.1 variant for the *IFNAR2* interferon receptor gene. Five of the sequences were replicated in a meta-analysis of a similar number of hospitalised cases from the COVID-19 Host Genetics Initiative ([Nasir, Wolford and Veerapen, 2020](https://www.covid19hg.org/blog/2020-11-24-covid-19-hgi-results-for-data-freeze-4-october-2020/)).

All of these acronyms may appear cryptic, but they refer to genes that may be associated with the severe conditions of COVID-19. 

The researchers found that the two biological mechanisms that predispose to the severe form of infection are linked to 1) the body's response to invading viruses, and 2) the consequent inflammatory response of the host - i.e. patient. 


## Innate antiviral defences

In the study, [mendelian randomisation](https://www.cdc.gov/genomics/events/precision_med_pop.htm) revealed a causal link between COVID-19 severity and high expression of *TYK2*. The *TYK2* gene is associated with the infamous “*cytokine storm*,” an exaggerated immune response that can lead to life-threatening complications (such as acute respiratory distress syndrome and multiple organ failure).

The findings of the study are particularly significant as drugs capable of inhibiting the receptors identified by the researchers are already available. Thus, targeted therapeutic approaches could be developed for those patients who appear to be most at risk. For example, there are anti-TYK2 drugs for rheumatoid arthritis that could be tested, including a drug called Baricitinib. Of course, the “genetic profile” of each patient would need to be available to proceed in a targeted manner. The feasibility of obtaining this data should not be underestimated in terms of prevention at a population level.

Genetic findings also pointed towards a gene called *DPP9*, an enzyme involved in immune responses, and in a gene cluster called *OAS*, which helps prevent the virus from replicating. The *OAS1* finding was associated with increased susceptibility to SARS-CoV-2, whereas *DPP9* was associated with pulmonary fibrosis.


## Inflammatory processes

In the study, [mendelian randomisation](https://www.cdc.gov/genomics/events/precision_med_pop.htm) results also suggest a causal role for *IFNAR2*. Patients with severe COVID-19 had a lower expression of the gene *IFNAR2*, which encodes for the type I interferon receptor. Interferons are a family of cytokines with antiviral properties and act as an emergency signal to alert the immune system of an intruder or pathogen, such as bacteria or viruses.

Lower levels of interferon may lead to increased viral replication, and thus to a more severe presentation of the disease. Other studies have also implicated interferons in COVID-19 cases. Patients with variants in interferon-related genes ([Zhang et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32972995/)), or neutralising autoantibodies against interferons ([Bastard et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32972996/)) are more likely to have life-threatening COVID-19.

Based on these and other compelling data, interferons have been a target for researchers hoping to develop a COVID-19 treatment. Despite this, in a recent clinical trial, administering interferons had little or no effect on hospitalised patients with COVID-19 ([WHO Solidarity Trial Consortium, 2020](https://pubmed.ncbi.nlm.nih.gov/33264556/)). Moreover, a relatively large study failed to replicate the association of variants in interferon genes with severe COVID-19 ([Povysil et al., 2020](https://pubmed.ncbi.nlm.nih.gov/33398295/)).


## Socio-economic factors: Another piece to the puzzle

One of the hallmarks of COVID-19 is its disproportionate effect on Black, Asian and minority ethnic (BAME) communities, including frontline healthcare workers ([https://www.bmj.com/covid-memorial](https://www.bmj.com/covid-memorial)). According to figures reported by the British Medical Association, BAME workers accounted for 63% of deaths among healthcare workers but account for only 21% of staff, with 64% of deaths among BAME nurses who account for 20% of nurses, and 95 % of deaths amongst BAME doctors who represent 44% of doctors ([Shaw et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32949501/)).


There are multiple reasons for the high mortality rates in distressed communities, such as age, number of chronic ailments per capita, and socio-economic status. Italy was a case in point, demonstrating that age is a strong predictor of mortality from COVID-19 ([Onder, Rezza and Brusaferro, 2020](https://pubmed.ncbi.nlm.nih.gov/32203977/)).

Even after accounting for sex, age and deprivation, death rates from COVID-19 for Indians, Pakistanis, and Bangladeshis in the UK is up to double that of their white counterparts. Similar trends have been reported in the U.S, and other parts of the world. According to the WHO, poverty, overcrowded housing, and poor education contribute to the health inequalities.

In a study investigating the associations between socio-economic status and COVID-19–related cases (n=1,089,999) and mortality (n=62,298) in the U.S, lower education level (relative risk (RR) 1.10) and areas with a higher proportion of Black residents (RR 1.01) account for a disproportionately higher number of deaths ([Hawkins, Charles and Mehaffey, 2020](https://pubmed.ncbi.nlm.nih.gov/33227595/)). Similar results were found in another study, where Black Americans were more vulnerable to COVID-19 than other ethnic groups ([Abedi et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32875535/)).


## Conclusion

Numerous studies have shown the importance of genes involved in host cell viral entry and immune responses. Continued collaborative efforts are essential to elucidate the role of host genetic factors that determine COVID-19 severity and susceptibility. While the search for genetic risk factors are underway, it is equally important to investigate the socio-economic factors underpinning health inequalities. Genetics may give us new therapeutic opportunities, but it alone will not lead to an equitable fight against COVID-19.

As we enter the New Year, there is a glimmer of light at the end of the pandemic tunnel. With the development and distribution of vaccines coming to fruition, we should hope to see a dramatic decrease in the numbers of COVID-19 cases.


## Acknowledgements

Thank you to Andrea Ganna for thoughtful feedback and revisions. We would especially like to acknowledge Sophie Limou for contributing to the creation of Figure 2.


## References
Abedi, V., Olulana, O., Avula, V., Chaudhary, D., Khan, A., Shahjouei, S., Li, J. and Zand, R., 2020. Racial, Economic, and Health Inequality and COVID-19 Infection in the United States. *Journal of Racial and Ethnic Health Disparities*.

Bastard, P., Rosen, L., Zhang, Q., Michailidis, E., Hoffmann, H., Zhang, Y., Dorgham, K., Philippot, Q., Rosain, J., Béziat, V., Manry, J., Shaw, E., Haljasmägi, L., Peterson, P., Lorenzo, L., Bizien, L., Trouillet-Assant, S., Dobbs, K., de Jesus, A., Belot, A., Kallaste, A., Catherinot, E., Tandjaoui-Lambiotte, Y., Le Pen, J., Kerner, G., Bigio, B., Seeleuthner, Y., Yang, R., Bolze, A., Spaan, A., Delmonte, O., Abers, M., Aiuti, A., Casari, G., Lampasona, V., Piemonti, L., Ciceri, F., Bilguvar, K., Lifton, R., Vasse, M., Smadja, D., Migaud, M., Hadjadj, J., Terrier, B., Duffy, D., Quintana-Murci, L., van de Beek, D., Roussel, L., Vinh, D., Tangye, S., Haerynck, F., Dalmau, D., Martinez-Picado, J., Brodin, P., Nussenzweig, M., Boisson-Dupuis, S., Rodríguez-Gallego, C., Vogt, G., Mogensen, T., Oler, A., Gu, J., Burbelo, P., Cohen, J., Biondi, A., Bettini, L., D'Angio, M., Bonfanti, P., Rossignol, P., Mayaux, J., Rieux-Laucat, F., Husebye, E., Fusco, F., Ursini, M., Imberti, L., Sottini, A., Paghera, S., Quiros-Roldan, E., Rossi, C., Castagnoli, R., Montagna, D., Licari, A., Marseglia, G., Duval, X., Ghosn, J., Tsang, J., Goldbach-Mansky, R., Kisand, K., Lionakis, M., Puel, A., Zhang, S., Holland, S., Gorochov, G., Jouanguy, E., Rice, C., Cobat, A., Notarangelo, L., Abel, L., Su, H. and Casanova, J., 2020. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. *Science*, 370(6515), p.eabd4585.

Fu, L., Wang, B., Yuan, T., Chen, X., Ao, Y., Fitzpatrick, T., Li, P., Zhou, Y., Lin, Y., Duan, Q., Luo, G., Fan, S., Lu, Y., Feng, A., Zhan, Y., Liang, B., Cai, W., Zhang, L., Du, X., Li, L., Shu, Y. and Zou, H., 2020. Clinical characteristics of coronavirus disease 2019 (COVID-19) in China: A systematic review and meta-analysis. *Journal of Infection*, 80(6), pp.656-665.

Hawkins, R., Charles, E. and Mehaffey, J., 2020. Socio-economic status and COVID-19–related cases and fatalities. *Public Health*, 189, pp.129-134.

Hu, Z., Song, C., Xu, C., Jin, G., Chen, Y., Xu, X., Ma, H., Chen, W., Lin, Y., Zheng, Y., Wang, J., Hu, Z., Yi, Y. and Shen, H., 2020. Clinical characteristics of 24 asymptomatic infections with COVID-19 screened among close contacts in Nanjing, China. *Science China Life Sciences*, 63(5), pp.706-711.

Jiang, S., Du, L. and Shi, Z., 2020. An emerging coronavirus causing pneumonia outbreak in Wuhan, China: calling for developing therapeutic and prophylactic strategies. *Emerging Microbes & Infections*, 9(1), pp.275-277.

Kenney, A., Dowdle, J., Bozzacco, L., McMichael, T., St. Gelais, C., Panfil, A., Sun, Y., Schlesinger, L., Anderson, M., Green, P., López, C., Rosenberg, B., Wu, L. and Yount, J., 2017. Human Genetic Determinants of Viral Diseases. *Annual Review of Genetics*, 51(1), pp.241-263.

Nasir, J., Wolford, B. and Veerapen, K., 2020. COVID-19 Host Genetics Initiative. [online] Covid19hg.org. Available at: <https://www.covid19hg.org/blog/2020-11-24-covid-19-hgi-results-for-data-freeze-4-october-2020/> [Accessed 7 February 2021].

Onder, G., Rezza, G. and Brusaferro, S., 2020. Case-Fatality Rate and Characteristics of Patients Dying in Relation to COVID-19 in Italy. JAMA,.
Pairo-Castineira, E., Clohisey, S., Klaric, L., Bretherick, A., Rawlik, K., Pasko, D., Walker, S., Parkinson, N., Fourman, M., Russell, C., Furniss, J., Richmond, A., Gountouna, E., Wrobel, N., Harrison, D., Wang, B., Wu, Y., Meynert, A., Griffiths, F., Oosthuyzen, W., Kousathanas, A., Moutsianas, L., Yang, Z., Zhai, R., Zheng, C., Grimes, G., Beale, R., Millar, J., Shih, B., Keating, S., Zechner, M., Haley, C., Porteous, D., Hayward, C., Yang, J., Knight, J., Summers, C., Shankar-Hari, M., Klenerman, P., Turtle, L., Ho, A., Moore, S., Hinds, C., Horby, P., Nichol, A., Maslove, D., Ling, L., McAuley, D., Montgomery, H., Walsh, T., Pereira, A., Renieri, A., Shen, X., Ponting, C., Fawkes, A., Tenesa, A., Caulfield, M., Scott, R., Rowan, K., Murphy, L., Openshaw, P., Semple, M., Law, A., Vitart, V., Wilson, J. and Baillie, J., 2020. Genetic mechanisms of critical illness in Covid-19. *Nature*.

Povysil, G., Butler-Laporte, G., Shang, N., Weng, C., Khan, A., Alaamery, M., Nakanishi, T., Zhou, S., Forgetta, V., Eveleigh, R., Bourgey, M., Aziz, N., Jones, S., Knoppers, B., Scherer, S., Strug, L., Lepage, P., Ragoussis, J., Bourque, G., Alghamdi, J., Aljawini, N., Albes, N., Al-Afghani, H., Alghamdi, B., Almutair, M., Mahmoud, E., Safie, L., Bardisy, H., Harthi, F., Alshareef, A., Suliman, B., Alqahtani, S., AlMalik, A., Alrashed, M., Massadeh, S., Mooser, V., Lathrop, M., Arabi, Y., Mbarek, H., Saad, C., Al-Muftah, W., Badji, R., Thani, A., Ismail, S., Gharavi, A., Abedalthagafi, M., Richards, J., Goldstein, D. and Kiryluk, K., 2020. Failure to replicate the association of rare loss-of-function variants in type I IFN immunity genes with severe COVID-19.

Qin, C., Zhou, L., Hu, Z., Zhang, S., Yang, S., Tao, Y., Xie, C., Ma, K., Shang, K., Wang, W. and Tian, D., 2020. Dysregulation of Immune Response in Patients With Coronavirus 2019 (COVID-19) in Wuhan, China. *Clinical Infectious Diseases*, 71(15), pp.762-768.

Severe Covid-19 GWAS Group, 2020. Genomewide Association Study of Severe Covid-19 with Respiratory Failure. New England Journal of Medicine, 383(16), pp.1522-1534.
Shaw, A., Flott, K., Fontana, G., Durkin, M. and Darzi, A., 2020. No patient safety without health worker safety. The Lancet, 396(10262), pp.1541-1543.
Wang, F., Huang, S., Gao, R., Zhou, Y., Lai, C., Li, Z., Xian, W., Qian, X., Li, Z., Huang, Y., Tang, Q., Liu, P., Chen, R., Liu, R., Li, X., Tong, X., Zhou, X., Bai, Y., Duan, G., Zhang, T., Xu, X., Wang, J., Yang, H., Liu, S., He, Q., Jin, X. and Liu, L., 2020. Initial whole-genome sequencing and analysis of the host genetic contribution to COVID-19 severity and susceptibility. *Cell Discovery*, 6(1).

WHO Solidarity Trial Consortium, 2020. Repurposed Antiviral Drugs for Covid-19 — Interim WHO Solidarity Trial Results. *New England Journal of Medicine*.

Zhang, Q., Bastard, P., Liu, Z., Le Pen, J., Moncada-Velez, M., Chen, J., Ogishi, M., Sabli, I., Hodeib, S., Korol, C., Rosain, J., Bilguvar, K., Ye, J., Bolze, A., Bigio, B., Yang, R., Arias, A., Zhou, Q., Zhang, Y., Onodi, F., Korniotis, S., Karpf, L., Philippot, Q., Chbihi, M., Bonnet-Madin, L., Dorgham, K., Smith, N., Schneider, W., Razooky, B., Hoffmann, H., Michailidis, E., Moens, L., Han, J., Lorenzo, L., Bizien, L., Meade, P., Neehus, A., Ugurbil, A., Corneau, A., Kerner, G., Zhang, P., Rapaport, F., Seeleuthner, Y., Manry, J., Masson, C., Schmitt, Y., Schlüter, A., Le Voyer, T., Khan, T., Li, J., Fellay, J., Roussel, L., Shahrooei, M., Alosaimi, M., Mansouri, D., Al-Saud, H., Al-Mulla, F., Almourfi, F., Al-Muhsen, S., Alsohime, F., Al Turki, S., Hasanato, R., van de Beek, D., Biondi, A., Bettini, L., D’Angio’, M., Bonfanti, P., Imberti, L., Sottini, A., Paghera, S., Quiros-Roldan, E., Rossi, C., Oler, A., Tompkins, M., Alba, C., Vandernoot, I., Goffard, J., Smits, G., Migeotte, I., Haerynck, F., Soler-Palacin, P., Martin-Nalda, A., Colobran, R., Morange, P., Keles, S., Çölkesen, F., Ozcelik, T., Yasar, K., Senoglu, S., Karabela, Ş., Rodríguez-Gallego, C., Novelli, G., Hraiech, S., Tandjaoui-Lambiotte, Y., Duval, X., Laouénan, C., Snow, A., Dalgard, C., Milner, J., Vinh, D., Mogensen, T., Marr, N., Spaan, A., Boisson, B., Boisson-Dupuis, S., Bustamante, J., Puel, A., Ciancanelli, M., Meyts, I., Maniatis, T., Soumelis, V., Amara, A., Nussenzweig, M., García-Sastre, A., Krammer, F., Pujol, A., Duffy, D., Lifton, R., Zhang, S., Gorochov, G., Béziat, V., Jouanguy, E., Sancho-Shimizu, V., Rice, C., Abel, L., Notarangelo, L., Cobat, A., Su, H. and Casanova, J., 2020. Inborn errors of type I IFN immunity in patients with life-threatening COVID-19. *Science*, 370(6515), p.eabd4570.

Zhao, J., Yang, Y., Huang, H., Li, D., Gu, D., Lu, X., Zhang, Z., Liu, L., Liu, T., Liu, Y., He, Y., Sun, B., Wei, M., Yang, G., Wang, X., Zhang, L., Zhou, X., Xing, M. and Wang, P., 2020. Relationship Between the ABO Blood Group and the Coronavirus Disease 2019 (COVID-19) Susceptibility. *Clinical Infectious Diseases*.
